libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -26,11 +27,16 @@ const AP_Param::GroupInfo AP_Vehicle::var_info[] = {
     // @Path: ../AP_VisualOdom/AP_VisualOdom.cpp
     AP_SUBGROUPINFO(visual_odom, "VISO",  3, AP_Vehicle, AP_VisualOdom),
 #endif
-
     // @Group: VTX_
     // @Path: ../AP_RCTelemetry/AP_VideoTX.cpp
     AP_SUBGROUPINFO(vtx, "VTX_",  4, AP_Vehicle, AP_VideoTX),
 
+#if HAL_WITH_FRSKY_TELEM_BIDIRECTIONAL
+    // @Group: CAM_RC_
```

Comment:
Parameter descriptions are *so* not right :-)

Reply:
fixed